CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 21.9
+We're dedicated to enhancing your WooCommerce app experience! We've ironed out the kinks related to discarding images during uploads in the Product Form. Plus, we've ramped up our login flow accessibility to make your journey smoother and more intuitive. Enjoy the fresh update and newly polished features!
```

Comment:
Can we maybe replace `kinks` with something simpler like `issues`?

Reply:
I agree. Done in https://github.com/woocommerce/woocommerce-ios/pull/15285/commits/10fb57df7f52a7dd22c706fcef085a410921ca00